scripts/india_lgd/local_government_directory_districts/LocalGovernmentDirectory_Districts.tmcf

```diff
@@ -0,0 +1,12 @@
+Node: E:LocalGovernmentDirectory_Districts->E0
+typeOf: dcs:Place
+lgdCode: C:LocalGovernmentDirectory_Districts->LGDStateCode
+
+Node: E:LocalGovernmentDirectory_Districts->E1
+typeOf: dcs:Place
```

Comment:
Here, do we expect to resolve all the nodes against DC?  If not, and we will create new nodes, then it might be better to provide a more precise type: AdministrativeArea2.

Reply:
Correct. New districts will be created as part of this PR. Will use `AdministrativeArea2`